spec/linter-stylelint-spec.js

```diff
@@ -7,117 +7,123 @@ describe('The stylelint provider for Linter', () => {
 
   beforeEach(() => {
     atom.workspace.destroyActivePaneItem();
-    waitsForPromise(() => {
-      atom.packages.activatePackage('linter-stylelint');
-      return atom.packages.activatePackage('language-css').then(() =>
-        atom.workspace.open(path.join(__dirname, 'fixtures', 'good.css'))
-      );
-    });
     atom.config.set('linter-stylelint.usePreset', true);
-  });
 
-  describe('checks bad.css and', () => {
-    let editor = null;
-    beforeEach(() => {
-      waitsForPromise(() => {
-        return atom.workspace.open(path.join(__dirname, 'fixtures', 'bad', 'bad.css')).then(openEditor => {
-          editor = openEditor;
-        });
+    waitsForPromise(() => {
+      return atom.packages.activatePackage('linter-stylelint').then(() => {
```

Comment:
Btw, for a plugin like this _just_ `atom.packages.activatePackage('linter-stylelint')` would be enough, the activation of the language will only be needed with `activationHooks` (It doesn't hurt anything though).